### DISCUSSION TITLE
Can't upload images - Error: Some files failed to upload

### DISCUSSION THREAD

**Discussion by jxs714** [formbricks/formbricks#2649] [General Requests]
Hey guys,

Im getting an error for my docker setup when trying to upload a logo or background image. The error says "Some files failed to upload
". I am using nginx with the following nginx directive

location ~ ^/ {
	proxy_pass http://localhost:3020;
	proxy_set_header Upgrade $http_upgrade;
	proxy_set_header Connection "upgrade";
	proxy_set_header Host $host;
	proxy_set_header X-Real-IP $remote_addr;
	proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
	proxy_set_header X-Forwarded-Proto $scheme;
	proxy_set_header X-Frame-Options SAMEORIGIN;
	proxy_buffers 256 16k;
	proxy_buffer_size 16k;
	client_max_body_size 50m;
	client_body_buffer_size 1m;
	proxy_read_timeout 600s;
	proxy_buffering off;
	proxy_cache off;
}

location ~* \.(css|js|jpg|jpeg|gif|png|ico|woff|woff2|ttf|otf|svg|eot)$ {
	proxy_pass http://localhost:3020;
	proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
	proxy_set_header Host $http_host;
	proxy_set_header X-Real-IP $remote_addr;
	proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
	proxy_set_header X-Forwarded-Proto $scheme;
}

how can I fix this issue?

**Comment by mattinannt** [comment]
@jxs714 Were you already able to solve the issue? Please join our Discord at https://formbricks.com/discord and open a help request there; that's normally the fasted way to reach us and to solve your issue quickly ;-)

**Comment by Lx** [comment]
For others coming across this issue, a cheap and nasty fix [from the Discord](https://discord.com/channels/979077669410979880/1250064749467668524/1251098820591226880) in the short term is to adjust the permissions on the `uploads` directory:

```
sudo chmod -R 777 uploads
```

A more appropriate longer-term solution might be to enable users to change the running UID and GID via `docker-compose.yaml` file environment variables.  (If this is already available, it is unfortunately not currently documented anywhere.)
